apps/studio/stories/Checkbox.stories.tsx

```diff
@@ -1,34 +1,159 @@
 import { Meta, StoryObj } from "@storybook/react";
 
-import { Checkbox, Label } from "@kampus/ui";
-
-const labels = ["köpekleri çıkarmak", "alışverişe gitmek", "kitap okumak"];
+import { Button, Checkbox, Form, FormControl, FormDescription, FormField, FormItem, FormLabel, Label, toast, useForm, z } from "@kampus/ui";
+import { Link } from "lucide-react";
+import { zodResolver } from "@hookform/resolvers/zod"
 
 const meta = {
   title: "Checkbox",
+  tags: ["autodocs"],
   component: Checkbox,
+  argTypes: {
+    disabled: {
+      control: "boolean",
+    },
+    checked: {
+      control: "boolean",
+    }
+  },
+
 } satisfies Meta<typeof Checkbox>;
 
 export default meta;
-type NewType = StoryObj<typeof meta>;
+type Story = StoryObj<typeof meta>;
+
+export const DefaultChecked = {
+  args: {
+    defaultChecked: true,
+  },
+} satisfies Story;
+
+export const OnFocus = {
```

Comment:
@usirin if you can take a look at this and show a pointer to handle this correctly, that would be awesome! 

-- we shouldn't manipulate DOM and focus is not doing what it meant to do, so this needs to be addressed but lets wait an answer from @usirin first

Reply:
@melihcanclk Easiest would be creating a component inside this file and use state with proper onFocus handlers over there

Reply:
I tried to remove component inside OnFocus story and implement a component that makes handling with useState hook but realize that i have to give onFocus function outside of the component so to change the state inside onFocus function, i shouldn't make changes in Checkbox props, meaning it should take {...props} and with changing args to that component, we should do the change but this is my idea. If there is any way to change text inside the component by giving useState from args, I can implement it but this is what i did
 
![Screenshot from 2023-08-26 11-04-15](https://github.com/kamp-us/monorepo/assets/42675463/18c08c8a-f34c-489e-b53b-c80cc08c6e47)

I can commit this or I can remove the whole onFocus part either because it really doesn't metter, why do i think in this way is that in Shadcn UI components' examples, there is no `onFocus` example. It has only With Text, Disabled and Form examples where all are implemented already, and if this is ok, i can do one extra thing that i did think that i could be doing but i didn't because of this same reason which is changing color of background and border of the checkbox over table down below.

Reply:
I made some comments about this and I think the way you handled in the new version looks good to me! After you make those changes, we can review it again and merge this :) Thank you so much for the hard work and thinking about additions to the regular use cases 🚀 💯

Reply:
I've tried to fix those and I'll be waiting for your feedback, thank you for your comments dear @cansirin 💜.